packages/jest-serializer-merge-styles/jest.config.js

```diff
@@ -2,7 +2,7 @@ const path = require('path');
 const { createConfig } = require('@fluentui/scripts/jest/jest-resources');
 
 const config = createConfig({
-  snapshotSerializers: [path.resolve(__dirname)],
+  snapshotSerializers: [path.join(path.resolve(__dirname), 'src', 'index.ts')],
```

Comment:
DX: jest will directly resolve typescript file so there is no need to build prior to run tests